test/micro/org/openjdk/bench/java/util/concurrent/Maps.java

```diff
@@ -106,6 +112,21 @@ public void testConcurrentHashMap() {
         position = pos;
     }
 
+    @Benchmark
+    public ConcurrentHashMap<Integer, Integer> testConcurrentHashMapCopyConstructor() {
+        return new ConcurrentHashMap<>(staticMap);
+    }
+
+    @Benchmark
+    public ConcurrentHashMap<Integer, Integer> testConcurrentHashMapPutAll() {
+        ConcurrentHashMap<Integer, Integer> map = new ConcurrentHashMap<>();
```

Comment:
I think this benchmark could be made more accurate by creating the new, temporary map with the right initial size (i.e. `ConcurrentHashMap<>(nkeys)`) to avoid calls to `tryPresize()` in this setup step.

Reply:
I updated. The numbers are surprisingly the same. I guess the benchmark compute time is dominated by putAll().

Reply:
OK, thanks nevertheless.